api/rest/config.go

```diff
@@ -185,7 +206,7 @@ func (cfg *Config) Default() error {
 	cfg.CORSExposedHeaders = DefaultCORSExposedHeaders
 	cfg.CORSAllowCredentials = DefaultCORSAllowCredentials
 	cfg.CORSMaxAge = DefaultCORSMaxAge
-
+	cfg.HTTPLogFile = ""
```

Comment:
Pay attention to context. You have attached the HTTPLogFile option to the bottom next to some CORS options. It would fit better after Auth. Also keep consistency with rest:

```
// Logs
cfg.HTTPLogFile = ""
```